src/GitVersionCore.Tests/Mocks/MockCommit.cs

```diff
@@ -2,6 +2,8 @@
 using System.Collections.Generic;
 using System.Diagnostics;
 using LibGit2Sharp;
+using Commit = GitVersion.Commit;
```

Comment:
I can't find `Commit.cs` to review its structure, am I blind for not seeing it?

Reply:
They are currently in one place in the GitObjects.cs. I put them all together so that I can easier do the refactoring